File: management/src/main/resources/static/controller/AcceptanceImportPassRejectionController.js
'use strict';
/*******************************************************************************
 * AcceptanceImportPassRejectionController
 ******************************************************************************/
app.controller('AcceptanceImportPassRejectionController', ['AcceptanceImportPassRejectionService', '$scope', function (AcceptanceImportPassRejectionService, $scope) {
  
  $scope.init = function () {
    console.log('Controller(js): init()');
    AcceptanceImportPassRejectionService.init();
    $scope.serviceName = AcceptanceImportPassRejectionService.getStorageData();
  }
  
}]);

File: management/src/main/java/io/hexaforce/management/controller/AcceptanceImportPassRejectionController.java
package io.hexaforce.management.controller;

import java.util.Arrays;
import java.util.List;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.http.HttpStatus;
import org.springframework.http.ResponseEntity;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.ResponseBody;
import org.springframework.web.bind.annotation.RestController;

import io.hexaforce.dievas.commons.DievasServerMesseage;
import io.hexaforce.dievas.service.AcceptanceImportPassRejectionService;
import lombok.extern.slf4j.Slf4j;

/**
 * AcceptanceImportPassRejection router function.
 * 
 * @version 1.0.0.BUILD-SNAPSHOT
 * @author T.Tantaka 
 */

@Slf4j
@RestController
@RequestMapping("/api")
public class AcceptanceImportPassRejectionController {
	
	@Autowired
	private AcceptanceImportPassRejectionService acceptanceImportPassRejectionService;
	
	@ResponseBody
	@GetMapping(value = "/acceptance-import-pass-rejection")
	public ResponseEntity<List<DievasServerMesseage>> init() {
		log.info("Controller(java): init()");
		String x = acceptanceImportPassRejectionService.init();
		return new ResponseEntity<List<DievasServerMesseage>>(Arrays.asList(new DievasServerMesseage(x)),HttpStatus.OK);
	}
	
}
